src/index.html

```diff
@@ -4,9 +4,364 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Philosopher&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+    
+      <label class="header__logo-href">
+        <a href="#home">
+          <img 
+            class="header__logo" 
+            src="images/logo.png" 
+            alt="logo"
+          >
+        </a>
+      </label>
+    
+      <nav class="nav-menu">
+        <ul class="nav-menu__list">
+          <li class="nav-menu__list-item">
+            <a class="is-active nav-menu__link" href="#">Apple</a>
+          </li>
+          <li class="nav-menu__list-item">
+            <a class="nav-menu__link" href="#">Samsung</a>
+          </li>
```

Comment:
add blanks between multiline elements
```suggestion
          <li class="nav-menu__list-item">
            <a class="is-active nav-menu__link" href="#">Apple</a>
          </li>
          
          <li class="nav-menu__list-item">
            <a class="nav-menu__link" href="#">Samsung</a>
          </li>
```